docs/tasksDir/pod-security-admission-with-kubewarden.md

```diff
@@ -1,50 +1,44 @@
 ---
-sidebar_label: "Using Pod Security Admission with Kubewarden"
-title: ""
+sidebar_label: "Pod Security Admission with KW"
+title: " Using Pod Security Admission with Kubewarden"
+description: Using Pod Security Admission with Kubewarden, since the Kubernetes 1.25 release.
+keywords: [kubewarden, pod security admission, pod security policy, kubernetes]
 ---
 
-# Using Pod Security Admission with Kubewarden
+Since Kubernetes 1.25, Pod Security Policies (PSP) have been removed.
+They're replaced by [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/) (PSA).
 
-Starting from Kubernetes 1.25 release, the Pod Security Policies have been removed
-and have been replaced by [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/),
-also known as *"PSA"*.
+PSA simplifies securing the Pods in Kubernetes clusters.
 
-Pod Security Admission has been created to simplify the process of securing Pods
-defined inside of Kubernetes clusters. Because of that it doesn't expose the
-same amount of configuration knobs provided by Pod Security Policies.
+PSA has three profiles for namespaces: `privileged`, `baseline`, and `restricted`.
 
-Pod Security Admission works at the Namespace level and provides three profiles
-that can be applied: `privileged`, `baseline` and `restricted`.
-The `privileged` profile has few limitations, while the `restricted` one is the
-one that tightens Pod permissions.
+The `privileged` profile has few limitations.
+The `restricted` profile tightens Pod permissions.
 
-In addition to choosing the profile to be used, the Kubernetes operators can decide
-what kind of action is going to be performed by Pod Security Admission controller
-once a violation is detected. The possible actions are: `enforce`, `audit` and
-`warn`.
+A PSA controller can perform actions when a violation is detected.
+The actions are: `enforce`, `audit`, and `warn` and can be configured.
 
-Pod Security Admission is not a full replacement for the old Pod Security Policies.
+<!--TODO:Still suffering these limitations? Next three paras may need update.-->
```

Comment:
Yes, those limitations may be there for a long time. We don't need to write this in the docs, but PSAs are minimal and limited in scope by design; the Kubernetes team doesn't want to add a sprawling API without direction. The idea of the policies are securities working group is to let a standard emerge from the community first (e.g: Kubewarden), then think about adding those functionalities to the Kubernetes API.

Maybe we could reword it to:

> At the time of writing, with Kubernetes 1.26, it suffers the following limitations:

Reply:
It applies also to 1.28

Reply:
Thanks.